### DISCUSSION TITLE
Auto-inserted bullet point has space after it - can that be disabled?

### DISCUSSION THREAD

**Discussion by rykugur** [nvim-neorg/neorg#1164] [Q&A]
Say I'm typing a note/list and have a header:

```
* Some cool thought
* 
```

When I press enter, a single asterisk is inserted as expected, however it also adds a space. It's a very minor thing, but I find it pretty annoying when I want to create a Nth level bullet point, I have to hit backspace. Is there a setting (either within the neorg plugin, some other plugin, or neovim) that is causing this?

I use neovim on both linux and macOS, with the [LazyVim environment](https://www.lazyvim.org/).

**Comment by max397574** [comment]
imo the best thing would be if it would still insert the space because that's quite useful (especially for lists) but would automatically remove the space when you type a dash

**Comment by pysan3** [comment]
Default vim keybind is overloaded in norg to increment / decrement levels. 

`<C-t>, <C-d>` in insert mode and `>>, <<` for normal, visual mode.
